storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst

```diff
@@ -13,14 +13,18 @@ master encryption key is stored in a keyring. |xtrabackup| supports two keyring
 plugins: ``keyring_file``, and ``keyring_vault``. These plugins are installed
 into the ``plugin`` directory.
 
-Implemented in |xtrabackup| version 8.0.25, adds support for the ``keyring_file`` component, which is part of the component-based infrastructure MySQL includes to extend the server capabilities. The component is stored in the ``plugin`` directory. 
+Percona XtraBackup 8.0.28-21 adds support for the Amazon Key Management Service (AWS KMS). AWS KMS is an encryption and key management service used in the cloud. The keys and functionality can be used for other AWS services or your applications that use AWS. 
 
-See a `comparison of keyring components and keyring plugins <https://dev.mysql.com/doc/refman/8.0/en/keyring-component-plugin-comparison.html>`__ for more information.
+No configuration is required to back up a server with AWS KMS-enabled encryption.
```

Comment:
Add a note about:

This feature is *tech preview* quality.

Reply:
updated